main.tf

```diff
@@ -229,7 +221,12 @@ resource "aws_launch_configuration" "bastion_launch_configuration" {
     "${aws_security_group.bastion_host_security_group.id}",
   ]
 
-  user_data = "${data.template_file.user_data.rendered}"
+  user_data = templatefile("${path.module}/user_data.sh", {
```

Comment:
If we set up a bastion stack with a pre-built or ansible-managed AMI we should consider disabling injecting user-data.

Reply:
Makes sense.  Can address in another PR if we choose to go down that route.